src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -199,19 +200,19 @@ export class DotEditLayoutDesignerComponent implements OnInit {
             });
     }
 
-    private createSidebarForm() {
+    private createSidebarForm(): DotLayoutSideBar {
```

Comment:
Do we need if / else here?

Reply:
not sure if `this.pageState.layout.sidebar` will be present all the time? anyway I'll use a ternary to remove that